src/upgrade-compatibility-tool/install.md

```diff
@@ -0,0 +1,125 @@
+1. The Software Engineer sees that there are several customized areas broken in the inventory and catalog modules and they also get a complexity score of X. See the [Developer]({{site.baseurl}}/upgrade-compatibility-tool/developer.html) guide for more information on the complexity score.
+1. With this information, the Software Engineer is able to understand the complexity of the upgrade and is able to relay this information back to the partner's Account Manager.
+1. The Account Manager creates a timeline and cost for the Magento upgrade, which allows them to get their manager's approval.
+1. With their manager's approval, the Software Engineer works on the required code modifications to fix the broken modules.
+1. The Software Engineer runs the Upgrade Compatibility Tool one more time with a Magento pre-release to ensure there are no new issues and that their code changes fixed the problems found during the beta phase.
+1. Everything check’s out and the Software Engineer pushes the code to a staging environment where regression tests confirm all tests are green, which allows them to release the latest Magento version to production the same day that the Magento pre-release is released.
+
+![Upgrade Compatibility Tool audience](img/audience-uct.png)
+
+### Contact Upgrade Compatibility Tool
+
+To connect with the Upgrade Compatibility Tool team:
+
+1. Contact us on the Engineering Slack channel [Upgrade Compatibility Tool](https://magentocommeng.slack.com/archives/C019Y143U9F).
+1. Send us an email at [uct@adobe.com](mailto:uct@adobe.com).
+
+See the [Resources]({{site.baseurl}}/community/resources/resources.html) page for more information.
+
+## Prerequisites
+
+See [prerequisites]({{site.baseurl}}/upgrade-compatibility-tool/prerequisites.html) for more information.
+
+{:.bs-callout-info}
+You can run the Upgrade Compatibility Tool in any operating system. There is no requirement to run the Upgrade Compatibility Tool where your Magento instance is located. It is necessary for the Upgrade Compatibility Tool to have access to the source code of the Magento instance. For example, you can install the tool on one server and point it at your Magento installation on another server.
+
+If you are running the Upgrade Compatibility Tool against a Magento instance with large modules and files, the tool might require a high amount of RAM, at least 2GB RAM.
+
+### Recommended actions
+
+Magento best practices recommend to avoid having 2 modules with the same name, if this happens the Upgrade Compatibility Tool shows a segmentation fault error.
+
+To avoid this segmentation fault error it is recommended to run the Upgrade Compatibility Tool to analyze each specific module independently including the option `-m`:
+
+```bash
+bin/uct upgrade:check /(magento_instance_path) --coming-version=2.4.1 -m /(magento_module_path)
```

Comment:
@danidelcar, I think the `-m` flag needs better explanation, probably best would be to use an (abstract) example of `magento_module_path`.
Like `app/code/SomeVendor/SomeModule` or `vendor/some-vendor/some-module` ?

Reply:
Also what's up with the 2 leading `/` ?